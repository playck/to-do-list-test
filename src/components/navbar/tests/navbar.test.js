import Navbar from "../navbar";
import renderer from "react-test-renderer";

describe("Navbar", () => {
  it("renders", () => {
    const component = renderer.create(<Navbar />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
