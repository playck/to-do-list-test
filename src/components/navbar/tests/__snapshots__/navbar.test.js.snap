// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders 1`] = `
<nav
  className="navbar"
>
  <span>
    To-Do-List
  </span>
</nav>
`;
